examples/js/build-ohlcv-bars.d.ts.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"build-ohlcv-bars.d.ts","sourceRoot":"","sources":["../ts/build-ohlcv-bars.ts"],"names":[],"mappings":""}
```

Comment:
I don't think we should generate d.ts and .map files for the examples 🤔

Reply:
I agree for the user it might be cleaner, and I tried to turn the declaration off, unfortunately in composite mode it doesn't allow to set declarion to false. The options I can think of is:
- We leave them
- We add them to the .gitignore so they don't show up in the repository
- I can remove the use of project references and put that part like before

Reply:
@pcriadoperez but now we're also generating the .map in the main js/ folder, not sure if this might have a impact on the users or not
![image](https://github.com/ccxt/ccxt/assets/43336371/ea01f412-e0c7-4a14-9701-5a6c4b5fc51c)

Reply:
@carlosmiei, I don't it should, in any case it would help the user IDE to map the JS to the typescript files

Reply:
@pcriadoperez maybe not, but usually the d.map are not used in production builds and it increases the package size.